Source/BusinessMomentsAdapters/Backend/read/entities/EntityQueries.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { Resolver, Query } from 'type-graphql';
+import { Entity } from './Entity';
+import { IMongoDatabase, graphRoot } from '@dolittle/vanir-backend';
+import { injectable } from 'tsyringe';
+
+@Resolver()
+@graphRoot('entities')
+@injectable()
+export class EntityQueries {
+
+    constructor(private readonly _mongoDatabase: IMongoDatabase) { }
```

Comment:
Vanir is setting up the mongodatabase object instance for me. While doing so it is also magically capturing tenante-id from the http request header. This is cool magic, but I think the error messages I get when tenant-id header is not explaining directly what's wrong.

Here is the error message when I query all entities via Graphql:

```
{
  "errors": [
    {
      "message": "Cannot inject the dependency \"_mongoDatabase\" at position #0 of \"EntityQueries\" constructor. Reason:\n    Missing resource configurations for '445f8ea8-1a6f-40d7-b2fc-796dba92dc44'",
      "locations": [
        {
          "line": 3,
          "column": 5
        }
      ],
      "path": [
        "entities",
        "allEntities"
      ],
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR",
        "exception": {
          "stacktrace": [
            "Error: Cannot inject the dependency \"_mongoDatabase\" at position #0 of \"EntityQueries\" constructor. Reason:",
            "    Missing resource configurations for '445f8ea8-1a6f-40d7-b2fc-796dba92dc44'",
            "    at /Users/gh/code/dolittle/Studio/node_modules/tsyringe/dist/cjs/dependency-container.js:290:23",
            "    at Array.map (<anonymous>)",
            "    at InternalDependencyContainer.construct (/Users/gh/code/dolittle/Studio/node_modules/tsyringe/dist/cjs/dependency-container.js:266:34)",
            "    at InternalDependencyContainer.resolve (/Users/gh/code/dolittle/Studio/node_modules/tsyringe/dist/cjs/dependency-container.js:105:33)",
            "    at Object.get (/Users/gh/code/dolittle/Studio/node_modules/@dolittle/vanir-backend/graphql/GraphQLSchema.ts:39:34)",
            "    at IOCContainer.getInstance (/Users/gh/code/dolittle/Studio/node_modules/type-graphql/dist/utils/container.js:39:26)",
            "    at /Users/gh/code/dolittle/Studio/node_modules/type-graphql/dist/resolvers/create.js:15:51",
            "    at field.resolve (/Users/gh/code/dolittle/Studio/node_modules/apollo-server-core/src/utils/schemaInstrumentation.ts:103:18)",
            "    at resolveField (/Users/gh/code/dolittle/Studio/node_modules/graphql/execution/execute.js:466:18)",
            "    at executeFields (/Users/gh/code/dolittle/Studio/node_modules/graphql/execution/execute.js:294:18)"
          ]
        }
      }
    }
  ],
  "data": {
    "entities": null
  }
}```

It should have given me a clue about the tenant-id missing from the header.

Here is my query

```
query {
  entities {
    allEntities {
      _id
    }
  }
}
```

Of course, this work very well when tenant id is added to the header. I suggest you fix this in Vanir, and then updated the Vanir dependency in this project.

Reply:
Good point. I've added an issue for this in Vanir: https://github.com/dolittle-entropy/vanir/issues/192

Reply:
@einari  nice!